_posts/2022-03-07-fine-dining-for-indices.md

```diff
@@ -0,0 +1,408 @@
+	"some_other_field": "This field is not in the mapping to exemplify Dynamic Mapping."
+}
+```
+
+What's leftover in `_source`?
+
+```
+GET exclude_index/_doc/xxxXxXXxXxXx
+{
+	...
+	"found" : true,
+	"_source" : {
+		"drink_me" : "potion of index familiarity",
+		"eat_me" : "iron ration"
+	}
+}
+```
+
+The `_source` is filtered down to only the elements described in the `include` and `exclude` stanzas in the index mapping.
+
+
+## Nutritional Information Available
+
+If you're worried about the size of your index, don't forget that you can always check the amount of disk space its using with an API call. Need a fun at-home experiment? Try defining two exact indices, one with `_source` and one without. How does their resource usage compare? 
+
+Here's how our examples are adding up so far: 
+
+
+```
+GET /_cat/indices/dynamic_mapping_index?format=json
+[
+  {
+    "health" : "green",
+    "status" : "open",
+    "index" : "dynamic_mapping_index",
+    "uuid" : "9g7x9PRoSyGcQp-3UMrP6g",
+    "pri" : "1",
+    "rep" : "1",
+    "docs.count" : "1",
+    "docs.deleted" : "0",
+    "store.size" : "10.1kb",
+    "pri.store.size" : "5kb"
+  }
+]
+
+
+GET _cat/indices/explicit_mapping_index?format=json
+[
+  {
+    "health" : "green",
+    "status" : "open",
+    "index" : "explicit_mapping_index",
+    "uuid" : "D66KzvYJSme5WZQU28YQDw",
+    "pri" : "1",
+    "rep" : "1",
+    "docs.count" : "1",
+    "docs.deleted" : "0",
+    "store.size" : "8.3kb",
+    "pri.store.size" : "4.1kb"
+  }
+]
+
+GET _cat/indices/exclude_index?format=json
+[
+  {
+    "health" : "green",
+    "status" : "open",
+    "index" : "exclude_index",
+    "uuid" : "3T4shiqERUS5qJTDm6qTWQ",
+    "pri" : "1",
+    "rep" : "1",
+    "docs.count" : "1",
+    "docs.deleted" : "0",
+    "store.size" : "13.8kb",
+    "pri.store.size" : "6.9kb"
+  }
+]
+
+```
+
+
+## Flavor To Taste, and Bon Apetít!
+
+A lot of index functionality is controlled by these small, sometimes overlooked stanzas of configuration. Storing the source document, deciding whether to index or not index fields, or even whether new fields should be added on the fly are all things that have a lasting impact on the way it behaves. The way that OpenSearch comes out of the box is suitable for a lot of tasks, but there’s no replacement for having intimacy and forethought with your own data.  Getting comfortable with the index configuration options will help ensure your cluster is ingesting properly. 
+
+## How Was The Service Today?
+
+As a community its important to know whether we’re advocating for all of the users. Please get involved and let us know if we’re providing what is needed to make the most out of OpenSearch. Whether via the [GitHub Project](https://github.com/opensearch-project), the [community forums](https://discuss.opendistrocommunity.dev/) or any of the other ways you can [connect with the community.](https://opensearch.org/connect.html) 
```

Comment:
'As a community it's important to know whether we’re' -> 'As a community its important to know the team is'

'Please get involved and let us know if we’re providing what is needed to make the most out of OpenSearch' -> 'Please get involved and speak up if you find anything that would help you get more out of OpenSearch.'